rules/check-gtfo-bins/raw.rego

```diff
@@ -0,0 +1,121 @@
+    cmd := container.command[_]
+    gtfo_cmds := {
+        "/bin/sh",
+        "/bin/bash",
+        "/bin/zsh",
+        "/bin/ksh",
+        "/bin/csh",
+        "/bin/tmux",
+        "/bin/screen",
+        "/usr/bin/script",
+        "/bin/expect",
+        "/usr/bin/awk",
+        "/usr/bin/base64",
+        "/usr/bin/basename",
+        "/usr/bin/busybox",
+        "/usr/bin/cat",
+        "/usr/bin/chmod",
+        "/usr/bin/chown",
+        "/usr/bin/crontab",
+        "/usr/bin/curl",
+        "/usr/bin/cut",
+        "/usr/bin/diff",
+        "/usr/bin/dir",
+        "/usr/bin/du",
+        "/usr/bin/env",
+        "/usr/bin/expand",
+        "/usr/bin/expect",
+        "/usr/bin/find",
+        "/usr/bin/fmt",
+        "/usr/bin/fold",
+        "/usr/bin/ftp",
+        "/usr/bin/gdb",
+        "/usr/bin/grep",
+        "/usr/bin/head",
+        "/usr/bin/iconv",
+        "/usr/bin/iftop",
+        "/usr/bin/irb",
+        "/usr/bin/jjs",
+        "/usr/bin/journalctl",
+        "/usr/bin/jrunscript",
+        "/usr/bin/ld.so",
+        "/usr/bin/less",
+        "/usr/bin/logsave",
+        "/usr/bin/lua",
+        "/usr/bin/mail",
+        "/usr/bin/mawk",
+        "/usr/bin/more",
+        "/usr/bin/mtr",
+        "/usr/bin/nawk",
+        "/usr/bin/nmap",
+        "/usr/bin/node",
+        "/usr/bin/od",
+        "/usr/bin/perl",
+        "/usr/bin/pg",
+        "/usr/bin/php",
+        "/usr/bin/pip",
+        "/usr/bin/pip3",
+        "/usr/bin/python",
+        "/usr/bin/rake",
+        "/usr/bin/ruby",
+        "/usr/bin/run-mailcap",
+        "/usr/bin/run-parts",
+        "/usr/bin/sed",
+        "/usr/bin/setarch",
+        "/usr/bin/socat",
+        "/usr/bin/soelim",
+        "/usr/bin/strace",
+        "/usr/bin/strings",
+        "/usr/bin/systemctl",
+        "/usr/bin/tac",
+        "/usr/bin/tail",
+        "/usr/bin/tar",
+        "/usr/bin/taskset",
+        "/usr/bin/tclsh",
+        "/usr/bin/tee",
+        "/usr/bin/telnet",
+        "/usr/bin/tftp",
+        "/usr/bin/time",
+        "/usr/bin/tmux",
+        "/usr/bin/top",
+        "/usr/bin/tshark",
+        "/usr/bin/unexpand",
+        "/usr/bin/unshare",
+        "/usr/bin/vi",
+        "/usr/bin/view",
+        "/usr/bin/vim",
+        "/usr/bin/wget",
+        "/usr/bin/whois",
+        "/usr/bin/wish",
+        "/usr/bin/xargs",
+        "/usr/bin/xxd",
+        "/usr/bin/yum",
+        "/usr/local/bin/chkrootkit",
+        "/usr/local/bin/jd-gui",
+        "/usr/local/bin/jd-cli",
+        "/usr/local/bin/rkhunter",
+        "/usr/local/bin/wireshark",
+        "/usr/sbin/adduser",
+        "/usr/sbin/arp",
+        "/usr/sbin/chroot",
+        "/usr/sbin/chsh",
+        "/usr/sbin/groupadd",
+        "/usr/sbin/groupdel",
+        "/usr/sbin/groupmod",
+        "/usr/sbin/grpck",
+        "/usr/sbin/ifconfig",
+        "/usr/sbin/kpartx",
+        "/usr/sbin/mount.nfs",
+        "/usr/sbin/mount.nfs4"
+    }
+    cmd == gtfo_cmds[_]
+    msg := {
```

Comment:
The `msg` variable should contain more information in order to give the final user a better overview of the problem.
Here's the struct we use internally with the available fields: https://github.com/kubescape/opa-utils/blob/master/reporthandling/datastructuresv1.go#L23.
Here's some example about how we use them: https://github.com/kubescape/regolibrary/blob/master/rules/container-hostPort/raw.rego

Reply:
Thanks! I updated it now. 
I don't know what alertScore should i provide!

Reply:
You can left it like so!